WHCFC_Frontend/src/app/components/mission/mission.component.html

```diff
@@ -0,0 +1,9 @@
+<div class="section white-background">
+    <div class="content right">
+      <h2>Mission</h2>
+      <p>Our mission is to promote health, fitness, and community spirit through football.</p>
+    </div>
```

Comment:
Similar to before, please update content with Joe's Google Doc for WHCFC's missions.